packages/test-helpers/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './isAddress'
```

Comment:
Here :point_down: 

Not sure if these should be a package or not. For now, it's just code that the other packages can import in their tests relatively.

Reply:
I think it’s good enough for now, we can make it a package whenever it becomes needed. :+1:

Reply:
We had a utility for `isAddress` that I decide to remove because ethers support it. I review and have both: [`isAddress`](https://docs.ethers.io/v5/api/utils/address/#utils-isAddress) and [`isBytes`](https://docs.ethers.io/v5/api/utils/bytes/#utils-isBytes).

Maybe we should use that?

Reply:
I was trying to avoid using a library for now. Having a helpers package funnels everything to one location. You can go there and see if its got what you need instead of requiring you to know that Ethers can do it.

Let's keep it, and in the future _use_ Ethers from within the helpers when this becomes a package.